mailpoet/assets/js/src/automation/sections/templates-section.tsx

```diff
@@ -0,0 +1,33 @@
+import { __ } from '@wordpress/i18n';
+import { Button } from '@wordpress/components';
+import { MailPoet } from '../../mailpoet';
+import { workflowTemplates } from '../templates/config';
+import { TemplateListItem } from '../templates/components/template-list-item';
+
+export function TemplatesSection(): JSX.Element {
+  const templates = workflowTemplates.slice(0, 3);
+
+  return (
+    <section className="mailpoet-automation-section mailpoet-section-templates">
+      <h2>{__('Start with essentials', 'mailpoet')}</h2>
```

Comment:
In Figma, I see "Explore essentials", or was that changed somewhere?

Reply:
current Figma is [this](https://www.figma.com/file/SAeK2GqSPbGsO7j5iQPT1j/Automations?node-id=1919%3A298068), right?
![image](https://user-images.githubusercontent.com/6458412/199194677-a6fa9581-196b-4e93-8653-68a2ec9e8a9f.png)

I think the screenshot in the issue is outdated.

Reply:
@websupporter Ahhh, there are both strings in Figma, depending on which of those screens you look. @NeosinneR which one is correct?

Reply:
[This](https://www.figma.com/file/SAeK2GqSPbGsO7j5iQPT1j/Automations?node-id=1919%3A298068) should be the latest version - so "Explore essentials".

Reply:
684aad2fe

Reply:
After rebase: [c92e827](https://github.com/mailpoet/mailpoet/pull/4461/commits/c92e82786150d9e77ad1a49a28bfad613100505f)